tests/python/pants_test/engine/test_fs.py

```diff
@@ -327,10 +330,10 @@ def test_merge_directories(self):
       scheduler = self.mk_scheduler(rules=create_fs_rules())
       (empty_snapshot, roland_snapshot, susannah_snapshot, both_snapshot) = (
           scheduler.capture_snapshots((
-            PathGlobsAndRoot(PathGlobs(("doesnotmatch",), ()), temp_dir),
-            PathGlobsAndRoot(PathGlobs(("roland",), ()), temp_dir),
-            PathGlobsAndRoot(PathGlobs(("susannah",), ()), temp_dir),
-            PathGlobsAndRoot(PathGlobs(("*",), ()), temp_dir),
+            PathGlobsAndRoot(PathGlobs(("doesnotmatch",), ()), native(str(temp_dir))),
```

Comment:
This usage is a bit confusing to me. Why would we use two method calls rather than `tobytes` or `text_type` or whatever more specific thing is appropriate?

Reply:
Good point. `text_type()` works and is much cleaner. I forgot it can serve as a function and not just a type.